ddtrace/llmobs/_llmobs.py

```diff
@@ -100,6 +100,8 @@ def __init__(self, tracer=None):
         forksafe.register(self._child_after_fork)
 
     def _child_after_fork(self):
+        if not self.enabled:
```

Comment:
Question about this - don't we still want to recreate the writers and configure the trace processors even if enabled is false? What if a user tries to enable the llmobs service after forking a previously disabled llmobs service? Without `writer.recreate()` wouldn't we have the same forking issue as before (wrong writer instance is being started)?

Reply:
Maybe we need to move this enabled check to after the writer workers are recreated, and ensure this check happens before the trace processor is added? (The trace processor should only be added if llmobs is enabled, but I think the writers need to be recreated even if disabled)

Reply:
yes this is definitely true if we permit users to `enable()` later 🤔. I wonder then if we should be only be adding the trace processor and writers on `enable()` then? WDYT?

Reply:
Just making sure I'm on the same page here - If we only add the trace processor (and writers) on `enable()`, you're proposing `_child_after_fork()` is only responsible for recreating the writer workers right? 

What about cases where llmobs is already enabled and then is forked? `_child_after_fork()` won't automatically add the trace processor, will we have to add an `enable()` step inside then?